addon/serializers/json-api-bagaaravel.js

```diff
@@ -0,0 +1,28 @@
+import { classify, underscore } from '@ember/string';
+import JSONAPISerializer from 'ember-data/serializers/json-api';
+
+export default JSONAPISerializer.extend({
+  /**
+   * Hooks
+   */
+
+  // Bagaaravel uses snake_case for attributes instead of kebab-case.
+  // 'attributeName' -> 'attribute_name'
+  keyForAttribute(key) {
```

Comment:
I know, but chances are real slim atm that Bagaaravel changes its implementation.